dhcpv6/option_userclass.go

```diff
@@ -38,16 +38,26 @@ func (op *OptUserClass) String() string {
 	return fmt.Sprintf("%s: [%s]", op.Code(), strings.Join(ucStrings, ", "))
 }
 
-// FromBytes builds an OptUserClass structure from a sequence of bytes. The
-// input data does not include option code and length bytes.
+// FromBytes builds an OptUserClass structure from a sequence of bytes.
+// The input data does not include option code and length bytes.
```

Comment:
Please add a comment here that explains that the function is able to parse both RFC-compliant options and MS-compatible, non-RFC-compliant options